draft-ietf-rats-eat.md

```diff
@@ -1494,8 +1494,8 @@ A profile can apply to Attestation Evidence or to Attestation Results or both.
 
 A profile document doesn't have to be in any particular format. It may be simple text, something more formal or a combination.
 
-In some cases CDDL may be created that replaces CDDL in this or other document to express some profile requirements.
-For example, to require the altitude data item in the location claim, CDDL can be written that replicates the location claim with the altitude no longer optional.
+In some cases it may be desirable for CDDL to be created that replaces CDDL in this or other document to express some profile requirements.
+For example, to require the altitude data item in the location claim, CDDL can be written that replicates the location claim with the altitude no longer optional.  In that case, the profile MUST define a new claim in the place of the original EAT claim and leverage the appropriate IANA registries to point to assign an appropriate claim identifier and pointer to the profile specification.
```

Comment:
If this means that we'd need to renumber the [nonce](https://datatracker.ietf.org/doc/html/draft-tschofenig-rats-psa-token#section-3.1.1) and the the [instance ID](https://datatracker.ietf.org/doc/html/draft-tschofenig-rats-psa-token#section-3.2.1) only because they express a constraint over the EAT type I am definitely opposed to this change.

Reply:
You are welcome to take your concerns to the Working Group and the IETF IoT Directorate and try to build consensus.  IANA registration requires a stable specification, and the nonce claim has been registered as per the current definition in the EAT document.